src/main/java/me/totalfreedom/totalfreedommod/util/FUtil.java

```diff
@@ -53,12 +57,32 @@
             ChatColor.LIGHT_PURPLE,
             ChatColor.YELLOW);
 
+    public static final List<ChatColor> CHAT_RAINBOW = Arrays.asList(
```

Comment:
Your rainbow nick command also does not work, as proved by myself and other admins. Your argument currently isnt holding up. I again, can see no reason for two of the same pools of colour.